plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -742,6 +742,11 @@ private void addSplitPluginDependencies(String thisPlugin, MavenRunner.Config mc
             Map<String,VersionNumber> toReplace = new HashMap<String,VersionNumber>();
             Map<String,VersionNumber> toAddTest = new HashMap<String,VersionNumber>();
             Map<String,VersionNumber> toReplaceTest = new HashMap<String,VersionNumber>();
+            overridePlugins.forEach(plugin -> {
```

Comment:
it crashes here with NPE if the option is not specified. Stacktrace from #97 

```
Analysis: coreDep=2.7.3 pluginDeps={} pluginDepsTest={jquery-detached=1.2.1, handlebars=1.1.1, bootstrap=1.3.2}
java.lang.NullPointerException
	at org.jenkins.tools.test.PluginCompatTester.addSplitPluginDependencies(PluginCompatTester.java:758)
	at org.jenkins.tools.test.PluginCompatTester.testPluginAgainst(PluginCompatTester.java:489)
	at org.jenkins.tools.test.PluginCompatTester.testPlugins(PluginCompatTester.java:278)
	at org.jenkins.tools.test.PluginCompatTesterCli.main(PluginCompatTesterCli.java:165)
Processing org.jenkins.tools.test.hook.TransformPom
Exception in thread "main" hudson.AbortException: Execution was aborted due to the failure in a plugin test (-failOnerror is set)
	at org.jenkins.tools.test.PluginCompatTester.testPlugins(PluginCompatTester.java:343)
	at org.jenkins.tools.test.PluginCompatTesterCli.main(PluginCompatTesterCli.java:165)
make: *** [demo-jdk11] Error 1

```

Reply:
https://github.com/jenkinsci/plugin-compat-tester/pull/97/commits/e1be928fa2d1d8f0dc739b4705b1a2f09eb2f86b would be a minimal fix